Assets/MixedRealityToolkit/Experimental/Providers/Hands/HandRay/TorsoDirectedHandRay.cs

```diff
@@ -0,0 +1,192 @@
+﻿using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Input
```

Comment:
Is there a reason this needs to be in MRTK Foundation folder and MRTK foundation packages? Could this instead go under `MRTK.Examples/AlternateHandRay` or `MRTK.SDK/AlternateHandRay`?

Otherwise, everybody that installs MRTK foundation would get this AltHandRay experimental feature. While it is quite interesting and simple, it might set a precedent for introducing larger components which could increase the size of the core MRTK package.

Perhaps we should amend our guidance for experimental features to mention that experimental features should go either in MRTK.SDK or MRTK.Examples unless absolutely necessary.

Reply:
It does not. I started off putting it next to the old hand ray implementation, and I sort of guessed where to put it after calling it experimental.

I don't think it exactly belongs in Examples, but SDK might make sense.